app/models/poll.rb

```diff
@@ -1,4 +1,13 @@
 class Poll < ActiveRecord::Base
+  include HasPublicAuthor
+  include Flaggable
+  acts_as_votable
+
+  acts_as_paranoid column: :hidden_at
```

Comment:
There is no requirement to hide Polls.

Reply:
If you remove either the column or the association on polls model, it raises an error `ActionView::Template::Error (undefined local variable or method `with_hidden' for #<Poll::ActiveRecord_Relation:0x007ff01d39d0b8>):`

Reply:
If you remove the association, it raises an error 
```ruby
ActionView::Template::Error (undefined local variable or method `with_hidden' for #<Poll::ActiveRecord_Relation:0x007ff01d39d0b8>):
```

Reply:
Remove both:
```
  acts_as_paranoid column: :hidden_at
  include ActsAsParanoidAliases
```

`add_column :polls, :hidden_at, :datetime` from migration